test/mysql-chassis.js

```diff
@@ -316,6 +326,10 @@ describe('mysql-chassis', () => {
 
   // })
 
+  // describe('transaction methods', () => {
+
```

Comment:
It's pretty simple, actually. You can look at [L129](https://github.com/bradwestfall/mysql-chassis/blob/master/test/mysql-chassis.js#L129) for an example of how to instrument the tests. Basically, just tell Sinon how to resolve on each call in the tests.